raiden/tests/scenarios/open-channel-happy-scenario.yaml

```diff
@@ -0,0 +1,55 @@
+settings:
```

Comment:
What's the goal of this scenario? Everything it tests is alreday tested in the other scenario. I would just remove it.

Reply:
Ok will remove it.